common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -350,7 +351,7 @@ def get_html(self):
             'cdn_eval': cdn_eval,
             'cdn_exp_group': cdn_exp_group,
             'id': self.location.html_id(),
-            'display_name': self.display_name_with_default_escaped,
+            'display_name': self.display_name_with_default,
```

Comment:
@robrap I am re-deploying this change to my sandbox, and will check it before merging.

Thanks for your help!